src/data/system_prompts.py

```diff
@@ -0,0 +1,34 @@
+def get_general_prompt():
+    return (
+        "You are an autonomous driving AI assistant. You receive a single image from the front camera. "
```

Comment:
Would be great if we can already implement some approach oriented system prompt option here as well. I.e. a switch between the different prompt types resulting in different system prompts. That way we have granular control, a history, and wont miss setting the correct system prompt for an approach.
Also: How did we miss changing the prompts here? This prompt doesnt fit the grids at all 😭

Reply:
Would be great if we can make this part of this PR. Otherwise we might forget setting the prompts accordingly again.

If there are too many PRs for you to work on currently, I can steal this from you.